### ISSUE TITLE
Touch Responsiveness on Buttons (Globally through the app)

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/MentorApp#78]
The touch area of buttons is not responsive to a large enough area to be "usable".
This can be most clearly seen on the hamburger icon and on the skip question functionality. 
We are visibly seeing users struggling when interacting with these buttons.
Please can you review and enlarge the area responsive to touch (should this be the issue)

**Comment by Danman7** [comment]
I want to help with this, but I can't get a clear idea of what the bug is about. The touch ('responsive') area is as large as the buttons, which are as large as the inVision design. If you want the touch area enlarged we should make the buttons bigger.

Yes, I can make the real buttons larger and invisible and display fake buttons inside with the same size, but that would take time and is a bad approach as the areas might interfere with each other. Generally, the size of the area should be as large as the visual button.

I've again tested the burger icon with both the old KitKat and on a new Oreo device. The size of the icon depends not only on the size of the screen, but on the resolution as well. I can easily press the burger icon and get the menu going. This doesn't mean there is no issue, but `We are visibly seeing users struggling when interacting with these buttons.` doesn't tell me much from my view how I can help.

**Comment by stacylorraineprendeville** [comment]
I am working on the same device for all testing
Testing on Samsung Galaxy Tab A6.
The hamburger menu icon is quite insensitive - normally I find I am "touching" 2 or 3 times to activate this. Also the back arrow button during the life map process.
I DO NOT have the issue on buttons such as create a life map or continue - These seem generally responsive to touch. I have watched 2 other people struggle on the same device with this issue.

As with regards to the buttons being bigger  I would prefer to have the fully responsive design in place before we start to revise the design. There is a separate issue for this.

**Comment by stacylorraineprendeville** [comment]
@gigerbytes @pA1nD anyone else seeing this issue?

**Comment by Danman7** [comment]
Fixed in bc3de4e0056683b1f76b80a0d987dcc12b742a09, 51a90514a2cb503714bf97c8ce332388430593d9, 21c9fb309919c58122c3a618900d32884050d00c
